From 54bc778625237704bf4c63d5266248c1cd492f41 Mon Sep 17 00:00:00 2001
From: Weiguang Kong <weiguang.kong@nxp.com>
Date: Wed, 28 Jun 2017 13:02:16 +0800
Subject: [PATCH 2036/5242] MLK-15296-2: include: uapi: add consumed cycles

commit  ffa3d69b4366414a59218f68408fcdf3fa4e18b8 from
https://source.codeaurora.org/external/imx/linux-imx.git

add a new structure element(cycles) to save the
consumed cycles that returned from dsp framework

Signed-off-by: Weiguang Kong <weiguang.kong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/uapi/linux/mxc_hifi4.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/mxc_hifi4.h b/include/uapi/linux/mxc_hifi4.h
index 7194290..b1d631b 100644
--- a/include/uapi/linux/mxc_hifi4.h
+++ b/include/uapi/linux/mxc_hifi4.h
@@ -43,6 +43,7 @@ struct decode_info {
 	void *out_buf_addr;
 	int   out_buf_size;
 	int   out_buf_off;
+	unsigned int cycles;
 };
 
 struct prop_info {
-- 
1.7.9.5

